packages/devtools_app/lib/src/screens/inspector_v2/layout_explorer/ui/utils.dart

```diff
@@ -441,3 +563,47 @@ class LayoutExplorerBackground extends StatelessWidget {
     );
   }
 }
+
+/// Builds and positions a label for the [LayoutExplorerBackground] as
+/// determined by the widget's padding.
+class PositionedBackgroundLabel extends StatelessWidget {
+  const PositionedBackgroundLabel({
+    super.key,
+    required this.labelText,
+    required this.labelColor,
+    required this.topPadding,
+    required this.bottomPadding,
+    required this.leftPadding,
+    required this.rightPadding,
+  });
+
+  final String labelText;
+  final Color labelColor;
+  final bool topPadding;
+  final bool bottomPadding;
+  final bool leftPadding;
+  final bool rightPadding;
```

Comment:
can we rename these to `hasTopPadding`, `hasBottomPadding`, etc. so that it is more evident that these are boolean values instead of doubles?

Reply:
Done!